openbas-model/src/main/java/io/openbas/database/repository/TagRepository.java

```diff
@@ -1,18 +1,23 @@
 package io.openbas.database.repository;
 
 import io.openbas.database.model.Tag;
-import jakarta.validation.constraints.NotNull;
+import org.jetbrains.annotations.NotNull;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface TagRepository extends CrudRepository<Tag, String>, JpaSpecificationExecutor<Tag> {
 
-    @NotNull
-    Optional<Tag> findById(@NotNull String id);
+  @NotNull
+  Optional<Tag> findById(@NotNull String id);
 
-    Optional<Tag> findByName(@NotNull String name);
+  @NotNull
+  Optional<Tag> findByName(@NotNull final String name);
+
```

Comment:
Maybe we could also add an ignoreCase here? Do we have a unique constraint on this column?

Reply:
We already have it :)

  @NotNull
  List<Tag> findByNameIgnoreCase(@NotNull final String name);